### PR TITLE
Grey-on-pink pull quotes and epics on DCR analysis

### PR DESCRIPTION
**Pull request by tkgnm**
## Why?

Trello ticket: https://trello.com/c/spYocfQB/621-grey-on-pink-pull-quotes-and-epics-dotcom-rendering

This PR tweaks the background colour of pull quotes and epics as the grey-on-pink may be hard to see. A similar change was implemented for tags on analysis articles - see [this PR](https://github.com/guardian/dotcom-rendering/pull/5747#pullrequestreview-1075952548)

## Screenshots

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://user-images.githubusercontent.com/102960825/195079264-af54e07c-4c0c-4b6b-9474-fcdc18590979.png 
[after]: https://user-images.githubusercontent.com/102960825/195392844-52824f3d-e06a-4371-844a-1cd6bb3b715a.png

### CODE REVIEW

**Comment by mxdvl** [comment]
I think your before and after screenshots are inverted.

**Comment by tkgnm** [comment]
> I think your before and after screenshots are inverted.

You're correct - fixed now.

**Review by AshCorr — APPROVED** [review]
Looks good!

File: dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx [hunk 1]
```diff
@@ -204,6 +204,54 @@ export const PullQuoteBlockComponent: React.FC<{
...
+								width: 25px;
+								height: 25px;
+								bottom: -25px;
+								position: absolute;
+								background-color: ${neutral[100]};
+							}
+						`,
+					]}
+				>
+					<QuoteIcon colour={palette.fill.quoteIcon} />
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx:235]: I think this colour here should match the background of the main square? It looks like there's different based on the screenshot

I think this file needs a little updating for this to work but ideally we'd have all this colour variation managed by decidePalette, and then we'd add any variation for `Design.Analysis` there

**Review by oliverlloyd — COMMENTED** [review]
Do you think it would work if we use `palette.fill.quoteIcon` for both quote background the the icon background?

I realise this is a bit of an additional refactor outside of your main goal (add a variation for Analysis) but maybe we can refactor things to rename `palette.fill.quoteIcon` to `palette.background.pullQuote` and then use it in both places?

File: dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx (hunk 1 above)
Comment (tkgnm) [reply dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx:235]: Thanks for the tips! Will action.

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 2]
```diff
@@ -1321,6 +1321,19 @@ const fillQuoteIcon = (format: ArticleFormat): string => {
 	return pillarPalette[format.theme].main;
 };
 
+const backgroundPullQuote = (format: ArticleFormat): string => {
+	switch (format.design) {
+		case ArticleDesign.Editorial:
+		case ArticleDesign.Letter:
+		case ArticleDesign.Comment:
+			return '#fbe6d5';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/lib/decidePalette.ts:1329]:
```suggestion
			/* TODO: Source foundation doesn't have this colour, once it does, remove the hex below */
			return '#fbe6d5';
```

**Review by oliverlloyd — APPROVED** [review]
Perfection

**Pull request merged**